extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -89,16 +90,47 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
       }
       return choices;
     };
-    // If choices need to be shuffled, shuffle them, otherwise order the
+    // If choices need to be shuffled, shuffle them, if question is answered
+    // once before, get the previous order of choices, otherwise order the
     // choices based on their original index.
-    this.choices = (
-      showChoicesInShuffledOrder.value ? shuffleChoices(choicesWithIndex) :
-      choicesWithIndex.sort((c1, c2) => c1.originalIndex - c2.originalIndex));
+    this.questionIsOnceAnswered = this.isQuestionOnceAnswered();
+    if (this.questionIsOnceAnswered) {
+      this.choices = this.getPreviousOrderOfChoices();
+    } else {
+      this.choices = (
+        showChoicesInShuffledOrder.value ? shuffleChoices(choicesWithIndex) :
+        choicesWithIndex.sort((c1, c2) => c1.originalIndex - c2.originalIndex)
+      );
+    }
     this.answer = null;
     this.currentInteractionService.registerCurrentInteraction(
       () => this.submitAnswer(), () => this.validityCheckFn());
   }
 
+  isQuestionOnceAnswered(): boolean {
+    // Check if choice-order attribute is present in question
+    // element which indicates that the question has been answered
+    // once previously.
+    let questionElement = document.querySelector(
+      '.oppia-rte-viewer.oppia-learner-view-card-top-content');
+    if (questionElement) {
+      if (questionElement.getAttribute('choice-order')) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
```suggestion
    return questionElement && questionElement.getAttribute('choice-order');
```

Reply:
making it like this was giving a type error as the this statement returns a string, but have modified it somewhat.

Reply:
That might be because `questionElement.getAttribute('choice-order')` returns a string. Just check the length and it will work:
```javascript
choiceOrder = questionElement.getAttribute('choice-order');
return questionElement && choiceOrder && choiceOrder.length > 0;
```

Reply:
@iamprayush It works but it will return null in case when choiceOrder does exists, but we only want boolean and even if I make it work, is there a issue with how it is currently implemented, as either way we have to create a if condition to assign choiceOrder if questionElement is present. I think the current way is also fine.

Reply:
Fair enough, thanks for checking!